recipes/openblas/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, CMake, tools
+import sys
+import os
+
+
+class OpenBLAS(ConanFile):
+    name = "openblas"
+    version = "0.3.7"
+    license = "BSD 3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openblas.net"
+    description = "An optimized BLAS library based on GotoBLAS2 1.13 BSD version"
+    topics = (
+        "OpenBLAS",
+        "BLAS",
+        "LAPACK"
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_lapack": [True, False]
+    }
+    default_options = {
+        "shared": True,
```

Comment:
```suggestion
        "shared": False,
```

By default, all packages libs are static.